index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>PedrinhoReiDelas</title>
+</head>
+<body>
+    <div>
+        <button>
+            <p>Hello
```

Comment:
Sugestão: fechar a tag do parágrafo.
```suggestion
            <p>Hello</p>
```